djhtml/tests/test_suite.py

```diff
@@ -0,0 +1,34 @@
+import os
+import unittest
+
+from djhtml.modes import HTML
+
+
+class TestSuite(unittest.TestCase):
+    DIR = os.path.join(os.path.dirname(__file__), "suite")
+
+    def test_available_files(self):
+        """
+        Loop over all the files in the suite directory and compare the
+        expected output to the actual output.
+
+        """
+        for filename in os.listdir(self.DIR):
+            with self.subTest(filename):
+                if filename.endswith(".in"):
+                    basename, _ = os.path.splitext(filename)
+                    with open(os.path.join(self.DIR, filename), "r") as f:
+                        inputfile = f.readlines()
+                    with open(os.path.join(self.DIR, f"{basename}.out"), "r") as f:
+                        expected_output = f.readlines()
+
+                    current_mode = HTML
```

Comment:
This code should be abstracted, I think.

```
def indent_lines(lines: List[str]) -> List[str]:
```

that way, it can be re-used by both the main code and the test suite.

Reply:
Yes, I know, but before I write that code I want to decide on the external interface of the `Mode` class. So for now I simply copied this bit from `__main__.py` :grin: